README.md

```diff
@@ -827,6 +827,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Cynthia Teeters](https://github.com/cynthiateeters)
   - [Cyril Baah](https://github.com/CyrilBaah)
   - [Céline Le Corvaisier](https://github.com/ocsiddisco)
+  - [Choa Choa](https://github.com/phuchoa2001)
```

Comment:
Please keep the document in the correct alphabetical order. Your name should be in line 807 instead of 830.

Reply:
I updated my pull request. Could you check it out?